integration/tests/bft/topicvalidator/orchestrator.go

```diff
@@ -0,0 +1,275 @@
+package topicvalidator
+
+import (
+	"fmt"
+	"sync"
+	"testing"
+
+	"github.com/rs/zerolog"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/exp/rand"
+
+	"github.com/onflow/flow-go/insecure"
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/model/hash"
+	"github.com/onflow/flow-go/model/messages"
+	"github.com/onflow/flow-go/network"
+	"github.com/onflow/flow-go/network/channels"
+	"github.com/onflow/flow-go/utils/logging"
+	"github.com/onflow/flow-go/utils/unittest"
+)
+
+const (
+	// numOfAuthorizedEvents the number of authorized events that will be created when the test orchestrator is initialized.
+	// The numOfAuthorizedEvents allows us to wait for a certain number of authorized messages to be received, this should
+	// give the network enough time to process the unauthorized messages. This ensures us that the unauthorized messages
+	// were indeed dropped and not unprocessed.
+	numOfAuthorizedEvents = 100
+
+	// numOfUnauthorizedEvents the number of unauthorized events per type to send by the test orchestrator.
+	numOfUnauthorizedEvents = 10
+)
+
+// testOrchestrator represents a simple testOrchestrator that passes through all incoming events.
+type testOrchestrator struct {
+	sync.Mutex
+	t                          *testing.T
+	logger                     zerolog.Logger
+	orchestratorNetwork        insecure.OrchestratorNetwork
+	codec                      network.Codec
+	unauthorizedEventsReceived flow.IdentifierList
+	authorizedEventsReceived   flow.IdentifierList
+	unauthorizedEvents         map[flow.Identifier]*insecure.EgressEvent
+	authorizedEvents           map[flow.Identifier]*insecure.EgressEvent
+	authorizedEventsWg         sync.WaitGroup
+	attackerAN                 flow.Identifier
+	attackerEN                 flow.Identifier
+	victimEN                   flow.Identifier
+	victimVN                   flow.Identifier
+}
+
+var _ insecure.AttackOrchestrator = &testOrchestrator{}
+
+func NewOrchestrator(t *testing.T, logger zerolog.Logger, attackerAN, attackerEN, victimEN, victimVN flow.Identifier) *testOrchestrator {
+	orchestrator := &testOrchestrator{
+		t:                          t,
+		logger:                     logger.With().Str("component", "bft_test_orchestrator").Logger(),
```

Comment:
```suggestion
		logger:                     logger.With().Str("component", "bft-test-orchestrator").Logger(),
```
To be more consistent with logging conventions.